index.html

```diff
@@ -896,6 +896,55 @@ <h4>Definitions</h4>
       </table>
     </section>
 
+    <section>
+      <h3><dfn data-lt="JsonWebKey2020" data-dfn-type="dfn" id="JsonWebKey2020">JsonWebKey2020</dfn></h3>
```

Comment:
-1 to the name change... Just please use JwkVerificationKey2020. JWK is the format, Verification Key is the purpose.

Reply:
Not when I want to use X25519 to as a keyAgreementKey. That's the problem is that I have to define a new type for each specific function when the same JWK key is usable for both ECDH functions and Signing functions.

Reply:
Yes, its been pointed out that keyAgreement and assertionMethod, might both be supported by a JsonWebKey2020 in the future...

I think the `publicKeyJwk` part retains the "verification" concept... without being needlessly verbose...

I recently chose to use the the term "EcdsaSecp256k1RecoveryVerificationMethod2020", because some verification methods use hashes of keys.... like `ethereumAddress`...

Long story short, a verificationMethod may be a key, it might be an identifier related to a key... its biasing against the use of JSON Web Keys, to call them verification keys, when they are just keys... I might use them for key agreement, and assertionMethod and capability invocation, I'm not sure its helpful to include "VerificationKey" everywhere.

Reply:
Disagree, see this comment for the reasoning: https://github.com/w3c/did-core-registries/pull/43#discussion_r422127142

Reply:
Gonna resolve this PR for now and we can move to discussions